tests/e2e/e2e_test.go

```diff
@@ -31,7 +31,7 @@ import (
 
 const (
 	fundAccountAmount = 10000000000
-	timeout           = 1 * time.Second
+	timeout           = 2 * time.Second
```

Comment:
Gross. Apparently this makes a difference.

Reply:
https://github.com/oasisprotocol/oasis-indexer/actions/runs/3201060501/attempts/1

Reply:
ahh I was going to comment on that line in the e2e PR! :D
Is 2s definitely enough? At what interval does the testnet produce blocks?

Reply:
> At what interval does the testnet produce blocks?

I believe it was 1 second.

> Is 2s definitely enough?

🙏 ? I think the `oasis-node` orchestration under the hood takes time and the indexer is pretty fast. We could bump it to 3s? 🤔